### PR TITLE
PLINK version 2.00a2.3LM

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+name = 'PLINK'
+version = '2.00a2.3LM-x86_64'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = 'Whole-genome association analysis toolset'
+
+toolchain = SYSTEM
+
+source_urls = ['https://s3.amazonaws.com/plink2-assets/alpha2']
+sources = ['plink2_linux_x86_64.zip']
```
Comment (Micket) [inline easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb:16]: Problematic filename. We'll need to differentiate them locally with the download_filename approach again.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'PLINK'
+version = '2.00a2.3LM-x86_64'
```
Comment (Micket) [inline easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb:8]: What is LM?

Comment (sassy-crick) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb]: Good question. I think when I started to look into it the version was called LM but that changed after 9.4. I done some changes regarding versioning as well.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb]: I used a previous script as a template but I agree, I should have spotted it and changed that. I hope that is better now.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/adbd58725ea81adc4d39b37255e97886 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**